io.github.nostar.DroidStar.yml

```diff
@@ -0,0 +1,40 @@
+app-id: io.github.nostar.DroidStar
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: droidstar
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-download
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - "*.a"
+  - "*.la"
+modules:
+  - name: imbe_vocoder
+    buildsystem: simple
+    build-commands:
+      - make
+      - install -Dm644 libimbe_vocoder.a /app/lib/libimbe_vocoder.a
+    sources:
+      - type: archive
+        url: https://github.com/nostar/imbe_vocoder/archive/refs/heads/master.zip
+        sha256: c9b55c55650e0d16301dcd2f04edbbd2130f1809aded5fc364f95ff28577a937
+  - name: droidstar
+    buildsystem: qmake
+    post-install:
+      - install -Dm644 images/droidstar.png /app/share/icons/hicolor/128x128/apps/io.github.nostar.DroidStar.png
+      - install -Dm644 com.github.nostar.DroidStar.desktop /app/share/applications/io.github.nostar.DroidStar.desktop
+    sources:
+      - type: archive
+        url: https://github.com/nostar/DroidStar/archive/refs/heads/main.zip
```

Comment:
I will repeat: no branch. Tricking it by downloading the `master` archive doesn't cut it. You need to pin a specific commit or tag.

Reply:
I will try to solve it .. I have more than 60 amateur radio programs to upload .. Have me patience

Reply:
Nice! :-)